README.md

```diff
@@ -100,8 +100,10 @@ We use `cabal's` [Nix-style local builds][nix] for development. To get started q
 git clone git@github.com:github/semantic.git
 cd semantic
 git submodule sync --recursive && git submodule update --init --recursive --force
+cabal update
```

Comment:
Should this be `new-update`?  That's what I've been running but maybe I've been misspelling it!

Reply:
You’re right, it should! Fixing.